src/status_im/ui/screens/chat/components/messages_skeleton.cljs

```diff
@@ -0,0 +1,93 @@
+(ns status-im.ui.screens.chat.components.messages-skeleton
+  (:require [status-im.ui.components.react :as react]
+            [quo.react-native :as rn]
+            [status-im.ui.components.animation :as animation]
```

Comment:
hi @smohamedjavid, Thank you for your work. Please can we use [reanimated](https://github.com/status-im/status-mobile/blob/develop/src/quo2/reanimated.cljs) for animations.

Reply:
Changed to Reanimated 👍